filters/include/pcl/filters/impl/sampling_surface_normal.hpp

```diff
@@ -59,7 +59,7 @@ pcl::SamplingSurfaceNormal<PointT>::applyFilter (PointCloud &output)
   PointCloud data = *input_;
   partition (data, 0, npts, min_vec, max_vec, indices, output);
   output.width = 1;
-  output.height = std::uint32_t (output.points.size ());
+  output.height = output.size ();
```

Comment:
height being set instead of width. Unsure if we're in the same code block

Reply:
Split this into a separate PR